lib/oxrdf/src/blank_node.rs

```diff
@@ -414,4 +444,33 @@ mod tests {
         assert_eq!(align_of::<BlankNode>(), 8);
         assert_eq!(align_of::<BlankNodeRef<'_>>(), 8);
     }
+
+    #[test]
+    #[cfg(feature = "serde")]
+    fn test_serde() {
+        let b = BlankNode::new_from_unique_id(0x42);
+        let json = serde_json::to_string(&b).unwrap();
+        assert_eq!(json, "{\"value\":\"42\"}");
+        let b2: BlankNode = serde_json::from_str(&json).unwrap();
+
+        // WARNING: this a lossy process as blank nodes will always
+        // be reconstructed as named
+        assert_eq!(b2, BlankNode::new("42").unwrap());
```

Comment:
FYI

Reply:
Is this true? The blank node ID `"42"` should be properly parsed as an anonymous one.